### DISCUSSION TITLE
Japanese House of Representatives map?

### DISCUSSION THREAD

**Discussion by Joe13132** [yapms/yapms#45] [Ideas]
Hello YAPMS, I would like to have this new feature into YAPms such as the Japanese House of Representatives from the 2021 election and a blank map of the Japanese House of Representatives. 

I would love to see it in YAPms if anyone wants to use it for Election predictions or just for fun. 

Thanks 
![B476AAFF-5B1F-4217-922A-828ACFF85998](https://user-images.githubusercontent.com/120739439/208141478-8611a53d-28d2-432b-8cc3-65e7da2ca0c8.png)
![971C9A5A-D4CF-4C61-9931-70A31E920EF4](https://user-images.githubusercontent.com/120739439/208141500-c70bee9e-b197-4935-b03f-6a10b90b4d8c.png)

Joseph

**Comment by Fish1** [comment]
Definitely possible. I think we're a long way off from adding non-usa maps at the moment.

Unless we can find some people dedicated to making maps.

**Reply by Jackess4334** [reply]
I can probably make maps

**Reply by Fish1** [reply]
Someone has a PR open for it. I think there are a few issues with it, if you just wanna clean it up. I think it still has the default HTML popups enabled?

https://github.com/yapms/yapms/pull/123

**Reply by Jackess4334** [reply]
Ok, I can clean it up, and fix some code to make it work accurately
